src/main/java/com/group12/LaunchInterceptorConditionCollection.java

```diff
@@ -0,0 +1,39 @@
+package com.group12;
+
+import com.group12.model.Point;
+
+import java.util.List;
+
+public class LaunchInterceptorConditionCollection {
+
+    /**
+     * LIC #0 checks if there exists at least one set of two consecutive data points that are a distance greater than
+     * the given <b>length</b>.
+     *
+     * @param points list of radar echos ({@link Point})
+     * @param length has to be greater than 0 (zero)
+     * @return true if two consecutive points are greater than the given length, false otherwise.
+     * @throws IllegalArgumentException is thrown if <b>length</b> is less than 0 (zero)
+     */
+    public boolean LIC0(List<Point> points, double length) throws IllegalArgumentException {
```

Comment:
I feel like it's better to indicate this is `length1` (cause there's also `length2`), either in variable name or the docstring? This could make it easier when we are passing parameters to these LICs later.

Reply:
You are correct, will change.